dashboard/test/ui/features/teacher_tools/script_overview.feature

```diff
@@ -70,3 +70,28 @@ Feature: Unit overview page
     And I wait until element ".uitest-end-of-lesson-header:contains(You finished Lesson 3!)" is visible
     And I reload the page
     And  element ".uitest-end-of-lesson-header:contains(You finished Lesson 3!)" is not visible
+
+  Scenario: Unit overview lesson plan
+    Given I create an authorized teacher-associated student named "Blake"
+    When I sign in as "Teacher_Blake"
+    And I am on "http://studio.code.org/s/csp3-2019?no_redirect=true"
+    And I click selector "#uitest-lesson-plan" once I see it
+    When I switch tabs
+    And I wait until current URL contains "curriculum.code.org/csp-19/unit3/1/"
+    
+
+  Scenario: Unit overview new lesson plan
+    Given I create an authorized teacher-associated student named "Blake"
+    When I sign in as "Teacher_Blake"
+    And I am on "http://studio.code.org/s/allthemigratedthings?no_redirect=true"
+    And I click selector "#uitest-lesson-plan" once I see it
+    When I switch tabs
+    And I wait until current URL contains "/s/allthemigratedthings/lessons/1"
+
+  Scenario: Unit overview student resources
+    Given I create an authorized teacher-associated student named "Blake"
+    When I sign in as "Teacher_Blake"
```

Comment:
did you want to sign in as "Blake" here?

Reply:
Oh my teacher brain was still thinking of the teacher view here- is it worth adding another test on behalf of the student?

Reply:
For the student resources, I figure it's more important that we check the student view rather than the teacher? We could also do both

Reply:
It turns out it's a different button for students vs teachers. I added the student test so they're both covered!